BookingsApi/BookingsApi/Controllers/JudiciaryPersonController.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using BookingsApi.Contract.Requests;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Queries;
+using BookingsApi.DAL.Queries.Core;
+using BookingsApi.Domain;
+using BookingsApi.Validations;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NSwag.Annotations;
+
+namespace BookingsApi.Controllers
+{
+    [Produces("application/json")]
+    [Route("judiciaryperson")]
+    [ApiController]
+    public class JudiciaryPersonController : Controller
+    {
+        private const string BulkItemErrorMessage = "Could not add or update external Judiciary user with External Id: {0}";
```

Comment:
Does this variable need to be at the class level if it's only required for the `BulkJudiciaryPersonsAsync` method?

Reply:
Done